jerry-core/api/jerry.c

```diff
@@ -323,7 +323,6 @@ jerry_parse (const jerry_char_t *source_p, /**< script source */
   if (JERRY_CONTEXT (jerry_init_flags) & JERRY_INIT_MEM_STATS_SEPARATE)
   {
     jmem_stats_print ();
-    jmem_stats_reset_peak ();
```

Comment:
I don't know. I don't think anybody ever used this, since after the new byte code was landed the majority of its functionality had been lost. Perhaps we should keep the constant as a hole filler for now, and drop it in the next release.